### PR TITLE
Raise warning if shell_plus not run in debug mode

### PR DESCRIPTION
**Pull request by skyforest**
### Summary
- This works both in a Jupyter Lab environment / notebook and at the command line.
- Covers usage of `shell_plus`
- You must have the following in installed_apps in the test project:
            "dj_notebook",
             "django_extensions",
- make sure you are using the package build on this specific branch in pip.

Examples:

`DEBUG = False`

Screenshot 2023-10-09 at 11 02 02 PM
Screenshot 2023-10-09 at 11 03 06 PM

### For Reviewers:
1. Is this what you had in mind?
2. Should the error message text be rewritten?
3. `shell` has not been worked on yet. This only works for `shell_plus` but solution should be similar.

### CODE REVIEW

File: src/dj_notebook/__init__.py [hunk 1]
```diff
@@ -46,3 +60,15 @@ def activate(settings: str, quiet_load: bool = True) -> Plus:
             plus.print()
 
     return plus
+
+
+def is_notebook():
```
Comment (pydanny) [inline src/dj_notebook/__init__.py:65]: I did not expect this to ever be called outside of a Jupyter notebook. Is that a thing for your team?

File: src/dj_notebook/management/commands/shell_plus.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from django_extensions.management.commands import shell_plus
```
Comment (pydanny) [inline src/dj_notebook/management/commands/shell_plus.py:1]: I think I did not define the scope of this correctly. It is not meant to warn off on django-extension's `shell_plus`, rather to detect if dj-notebook is being run in production.

**Review by pydanny — CHANGES_REQUESTED** [review]
Apologies...

I realize now that the ticket, #75 is ambiguous and you tried to get clarification. It is meant to determine if dj-notebook is run in production or not. Not to change behaviors of django-extensions (which is what this PR is about).

This pull request could readily be adapted as a change to django-extensions itself. And arguably belongs there and not in dj-notebook. I'm happy to help you make that contribution to that library.

For dj-notebook, I was thinking of something like this within the `activate` function:

```python
def activate():
   # snip
   if plus.settings.DEBUG == False:
       # log warning message to stdout
```

**Comment by skyforest** [comment]
@pydanny, you know, that does seem much more reasonable of an ask for this :) Well, it was interesting to implement this but I think it is better off being a learning experience than a potential feature request over at django-extensions. I'm going to close this PR and get a fresh branch for what is actually needed. Thanks!

**Pull request closed without merging**